Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "tinybmp"
+description = "No-std, low memory footprint BMP image loader"
+version = "0.2.3"
+authors = ["James Waples <james@wapl.es>"]
+edition = "2018"
+repository = "https://github.com/embedded-graphics/embedded-graphics/tree/master/tinybmp"
+documentation = "https://docs.rs/tinybmp"
+categories = ["embedded", "no-std"]
+keywords = ["graphics", "embedded", "bmp", "bitmap", "image"]
```

Comment:
Should we start adding an embedded-graphics keyword to all crates in the e-g ecosystem? This way we could add a link to https://crates.io/keywords/embedded-graphics in the README to list all e-g related crates. e-t and e-l already include this keyword.

Reply:
Sounds good to me. As there's a 5 keyword limit, I'll remove `embedded` and replace it with `embedded-graphics`.